tests/test_client.py

```diff
@@ -27,268 +25,338 @@ def get_doc(
 
     doc = {"uploader": "uploader_123"}
-    r3 = swg_index_client.create_blank_entry(doc)
+    res_3 = client.post("/index/blank/", json=doc, headers=user)
+    rec_3 = res_3.json
 
-    r = swg_index_client.list_entries()
-    assert len(r.records) == 3
+    data_grab = client.get("/index/")
+    data_list = data_grab.json
+    assert len(data_list["records"]) == 3
 
-    r = swg_index_client.list_entries(uploader="uploader_123", acl="null", authz="null")
+    data_by_acl_authz = client.get("/index/?uploader=uploader_123&acl=null&authz=null")
+    data_list = data_by_acl_authz.json
 
-    assert len(r.records) == 2
-    assert {r2.did, r3.did} == {r.records[0].did, r.records[1].did}
-    assert r.records[0].acl == []
-    assert r.records[1].acl == []
-    assert r.records[0].authz == []
-    assert r.records[1].authz == []
+    assert len(data_list["records"]) == 2
+    assert {rec_2["did"], rec_3["did"]} == {
+        data_list["records"][0]["did"],
+        data_list["records"][1]["did"],
+    }
+    assert data_list["records"][0]["acl"] == []
+    assert data_list["records"][1]["acl"] == []
+    assert data_list["records"][0]["authz"] == []
+    assert data_list["records"][1]["authz"] == []
 
 
-def test_get_empty_acl_authz_record_after_fill_size_n_hash(swg_index_client):
+def test_get_empty_acl_authz_record_after_fill_size_n_hash(client, user):
     """
     Test create blank record -> fill hash and size -> get record with empty or none
     acl/authz
     """
     # create the first blank record, update size, hashes and acl/authz
     doc = {"uploader": "uploader_123"}
-    r1 = swg_index_client.create_blank_entry(doc)
+    res_1 = client.post("/index/blank/", json=doc, headers=user)
+    rec_1 = res_1.json
     updated = {"size": 10, "hashes": {"md5": "8b9942cf415384b27cadf1f4d2d981f5"}}
-    did1 = r1.did
-    r1 = swg_index_client.update_blank_entry(r1.did, rev=r1.rev, body=updated)
-    r1 = swg_index_client.update_entry(
-        r1.did, rev=r1.rev, body={"acl": ["read"], "authz": ["read"]}
-    )
-    r1 = swg_index_client.get_entry(r1.did)
-    assert r1.acl == ["read"]
-    assert r1.authz == ["read"]
-    assert r1.did == did1
+    did_1 = rec_1["did"]
+    res_2 = client.put("/index/blank/" + did_1, headers=user, json=updated)
+    rec_2 = res_2.json
+    rev = rec_1["rev"]
+    body = {"acl": ["read"], "authz": ["read"]}
+    res_1 = client.put("/index/" + did_1 + "?rev=" + rev, headers=user, json=body)
+    rec_1 = res_1.json
+    res_1 = client.get("/index/" + rec_1["did"])
+    rec_1 = res_1.json
+    assert rec_1["acl"] == ["read"]
+    assert rec_1["authz"] == ["read"]
+    assert rec_1["did"] == did_1
 
     # create the second blank record, only update size hashes and urls
     doc = {"uploader": "uploader_123"}
-    r2 = swg_index_client.create_blank_entry(doc)
-    did2 = r2.did
+    res_2 = client.post("/index/blank/", json=doc, headers=user)
+    rec_2 = res_2.json
+    did_2 = rec_2["did"]
     updated = {
```

Comment:
this record is never actually updated even though the comment says it should be..? 🤔 can you add `client.put /index/blank/` for this record too?